### DISCUSSION TITLE
上手实战从0制作一个基于图谱的语音智能助手 - siwei.io

### DISCUSSION THREAD

**Discussion by giscus** [siwei-io/siwei-io.github.io#30] [Comments]
# 上手实战从0制作一个基于图谱的语音智能助手 - siwei.io

Hands-on Course: Setup a KGQA system from scratch with Nebula Graph, VueJS, Flask on K8s.

https://siwei.io/learn/nebula-101-siwi-kgqa/

**Comment by akisou** [comment]
hello

**Reply by wey-gu** [reply]
Hi this learning env is broken due to https://kubernetes.io/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/

**Reply by wey-gu** [reply]
If you are interested in this project, see https://github.com/wey-gu/nebula-siwi/

While this is a pre-LLM era project :)

**Comment by jerryHo123** [comment]
katacoda.com 拒绝了我们的连接请求。  这是为啥呢

**Comment by cxljkb1994** [comment]
katacoda.com 拒绝了我们的连接请求。 这是为啥呢
